src/app/cli/src/init/mina_run.ml

```diff
@@ -531,6 +531,14 @@ let setup_local_server ?(client_trustlist = []) ?rest_server_port
                 ~metadata:
                   [ ("uri", `String (Uri.to_string uri))
                   ; ("context", `String "rest_server")
+                  ; ( "request"
```

Comment:
Nit: it is a fun way to DDOS as we could send large requests and force the log (which require some computations).

Reply:
Well, that could happen before. This PR make it more severe. 

I think maybe we can have build disabled those debug logs?